### ISSUE TITLE
Project future

### ISSUE THREAD

**Issue by MekDrop** [mariusbuescher/node-composer#4]
I created issue from comment on my pull request:

> Sorry, it is still early though. I like your idea, but I am not sure if I will continue to maintain this project. Actually this hasn't been touched in years. If you are interested, you can continue maintaining it.

> _Originally posted by @mariusbuescher in https://github.com/mariusbuescher/node-composer/pull/2#issuecomment-719373183_

**Comment by MekDrop** [comment]
I don't think that this project needs much maintenance. Unlinke [thecodingmachine/nodejs-installer](https://github.com/thecodingmachine/nodejs-installer) your project works in all major cases good enough.

But if you really don’t want to oversee this project, I could take it to the [Imponeer organization](https://github.com/imponeer). At least few other people are there could help me to keep this project a life.

But yesterday my pull request where accepted into [jakoch/awesome-composer](https://github.com/jakoch/awesome-composer) about this your awesome project, I would happier if you could keep this repo and do some small maintenance here (like accepting few my pull requests)...

**Comment by mariusbuescher** [comment]
Yes, you might be right, it does not take too much of time. AFAIR there was also some kind of compatibility issue on Windows. Due to not having a machine, that does run Windows, I can't confirm this. :shrug: 

Maybe transferring the project to your organization is a good idea, because it may have also more visibility than on my private account and also it is possible to make organization of contributions easier.

**Comment by MekDrop** [comment]
I didn't detected anything that is bad on Window. If this only a one reason for you, probably I could make another pull request with some CI tests (written as GitHub Actions) for Linux/Windows/MacOS.
